fhir-model/src/test/java/com/ibm/fhir/model/test/TestUtil.java

```diff
@@ -117,12 +116,26 @@ public static JsonObject getRequestJsonObject(String method, String url) {
      * the specified patient via a subject attribute.
      */
     public static Observation buildPatientObservation(String patientId, String fileName) throws Exception {
-        // TODO review Reference id
         Observation observation = readLocalResource(fileName);
 
         observation = observation
             .toBuilder()
-            .subject(Reference.builder().reference(string("Patient/" + patientId)).build())
+            .subject(Reference.builder().reference("Patient/" + patientId).build())
+            .build();
+        return observation;
+    }
+
+    /**
+     * Loads an Observation resource from the specified file, then associates it with
+     * the specified patient via a subject attribute.
```

Comment:
Also mention it sets the Practitioner / performer.

Reply:
done